install.md

```diff
@@ -255,7 +257,9 @@ On Debian, Raspbian and Ubuntu distributions, [enable the Kubic project
 repositories](../README.md#installing-crio) and install the following packages:
 
 ```bash
-apt-get update -qq && apt-get install -y \
+. /etc/os-release
+apt-get update -qq && \
+apt-get install -y -t ${VERSION_CODENAME}-backports \
```

Comment:
I think we may want an explanation on what VERSION_CODENAME IS (maybe add it to the table above?)

Reply:
For Debian 10 the VERSION_CODENAME would be "buster".
Maybe I shall just split the instructions into Debian 10(buster) and others(because code name would be different and I don't have such systems at the moment. Any idea?

Reply:
I think the cleanest way is to change the table above so that there's a column "CODENAME" and all of the debian flavors have "buster" in it. Then, when we support debian 11, we can just extend the table with a different name

Reply:
as shown in http://ftp.us.debian.org/debian/dists/, there will be no backport repo for Debian unstable, I think we have to give different instruction for different versions in the end

Reply:
Another option is we could say "if you're using debian 10 or related distros, you may need to add buster-backports to install the required packages" and then update for 11 one day